server/server.go

```diff
@@ -36,8 +39,19 @@ func (s *Server) Version(ctx context.Context, req *pb.VersionRequest) (*pb.Versi
 
 // CreatePodSandbox creates a pod-level sandbox.
 // The definition of PodSandbox is at https://github.com/kubernetes/kubernetes/pull/25899
-func (s *Server) CreatePodSandbox(context.Context, *pb.CreatePodSandboxRequest) (*pb.CreatePodSandboxResponse, error) {
-	return nil, nil
+func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxRequest) (*pb.CreatePodSandboxResponse, error) {
+	var err error
+
+	// TODO: Parametrize as a global argument to ocid
+	ocidSandboxDir := "/var/lib/ocid/sandbox"
+	podSandboxDir := filepath.Join(ocidSandboxDir, req.GetConfig().GetName())
+
+	g := generate.New()
+
+	// TODO: Customize the config per the settings in the req
```

Comment:
Yes haven't forgotten :) I'll follow up  with TODO for all the steps. This was to get Haiyan unblocked for immediate next steps.

Sent from my iPhone

> On Jul 19, 2016, at 6:46 AM, Antonio Murdaca <notifications@github.com> wrote:
> 
> In server/server.go:
> 
> > @@ -36,8 +39,19 @@ func (s *Server) Version(ctx context.Context, req *pb.VersionRequest) (*pb.Versi
> >  
> >  // CreatePodSandbox creates a pod-level sandbox.
> >  // The definition of PodSandbox is at https://github.com/kubernetes/kubernetes/pull/25899
> > -func (s *Server) CreatePodSandbox(context.Context, *pb.CreatePodSandboxRequest) (*pb.CreatePodSandboxResponse, error) {
> > -	return nil, nil
> > +func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxRequest) (*pb.CreatePodSandboxResponse, error) {
> > +	var err error
> > +
> > +	// TODO: Parametrize as a global argument to ocid
> > +	ocidSandboxDir := "/var/lib/ocid/sandbox"
> > +	podSandboxDir := filepath.Join(ocidSandboxDir, req.GetConfig().GetName())
> > +
> > +	g := generate.New()
> > +
> > +	// TODO: Customize the config per the settings in the req
> the settings exposed by the image (?)
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>